frontend_tests/node_tests/unread.js

```diff
@@ -8,7 +8,8 @@
 // dependencies (except _).
 
 add_dependencies({
-    muting: 'js/muting.js'
+    muting: 'js/muting.js',
+    unread: 'js/unread.js'
```

Comment:
Node tests failed because `ReferenceError: unread is not defined` in muting.js. Simply, adding this solved the problem.